<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="504" started-at="2014-06-17T09:17:27Z" finished-at="2014-06-17T09:17:27Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="504" started-at="2014-06-17T09:17:27Z" finished-at="2014-06-17T09:17:27Z">
      <class name="com.ilabs.afv.testcases.LoginAFVAgentTestCase">
        <test-method status="FAIL" signature="testLoginAFVAgent(java.util.Hashtable)[pri:0, instance:com.ilabs.afv.testcases.LoginAFVAgentTestCase@35bcfced]" name="testLoginAFVAgent" duration-ms="108" started-at="2014-06-17T14:47:27Z" data-provider="testLoginAFVAgentDataProvider" finished-at="2014-06-17T14:47:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Browser1=ie}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Assertion failed due to the following reason : --> ]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Assertion failed due to the following reason : --> 
	at org.testng.Assert.fail(Assert.java:83)
	at com.ilabs.afv.testcases.LoginAFVAgentTestCase.testLoginAFVAgent(LoginAFVAgentTestCase.java:74)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.lang.NullPointerException
	at com.ilabs.afv.configuration.DriverConfiguration.initializeDriverOne(DriverConfiguration.java:50)
	at com.ilabs.afv.keywords.Keywords.launchBrowserOne(Keywords.java:43)
	at com.ilabs.afv.testcases.LoginAFVAgentTestCase.testLoginAFVAgent(LoginAFVAgentTestCase.java:43)
	... 24 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoginAFVAgent -->
        <test-method status="PASS" signature="closeBrowsers()[pri:0, instance:com.ilabs.afv.testcases.LoginAFVAgentTestCase@35bcfced]" name="closeBrowsers" is-config="true" duration-ms="1" started-at="2014-06-17T14:47:27Z" finished-at="2014-06-17T14:47:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowsers -->
      </class> <!-- com.ilabs.afv.testcases.LoginAFVAgentTestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
